src/main/kotlin/io/github/inductiveautomation/kindling/MainPanel.kt

```diff
@@ -139,59 +148,91 @@ class MainPanel : JPanel(MigLayout("ins 6, fill")) {
+                ) {
+                    val clipboard = Toolkit.getDefaultToolkit().systemClipboard
+                    if (clipboard.isDataFlavorAvailable(DataFlavor.stringFlavor)) {
+                        val clipString = clipboard.getData(DataFlavor.stringFlavor) as String
+                        openOrError(clipboardTool.title, "clipboard data") {
+                            clipboardTool.open(clipString)
+                        }
+                    } else {
+                        LOGGER.info("No string data found on clipboard")
+                    }
+                },
+            )
+        }
+    }
+
+    private val menuBar = JMenuBar().apply {
+        add(fileMenu)
+        add(pasteMenu)
         add(
-            JMenu("Paste").apply {
-                for (clipboardTool in Tool.tools.filterIsInstance<ClipboardTool>()) {
-                    add(
-                        Action(
-                            name = "Paste ${clipboardTool.title}",
-                        ) {
-                            val clipboard = Toolkit.getDefaultToolkit().systemClipboard
-                            if (clipboard.isDataFlavorAvailable(DataFlavor.stringFlavor)) {
-                                val clipString = clipboard.getData(DataFlavor.stringFlavor) as String
-                                openOrError(clipboardTool.title, "clipboard data") {
-                                    clipboardTool.open(clipString)
-                                }
-                            } else {
-                                LOGGER.info("No string data found on clipboard")
-                            }
-                        },
-                    )
+            JMenu("Help").apply {
+                add(debugMenu)
+                add(Action("Forum") {
+                    Desktop.getDesktop().browse(Kindling.forumThread)
+                })
+                if (!SystemInfo.isMacOS) {
+                    add(Action("About") {
+                        aboutDialog.isVisible = true
+                        aboutDialog.toFront()
+                    })
                 }
             },
         )
-        if (!SystemInfo.isMacOS) {
-            add(
-                JMenu("Preferences").apply {
-                    addMouseListener(
-                        object : MouseAdapter() {
-                            override fun mouseClicked(e: MouseEvent?) {
-                                preferencesEditor.isVisible = !preferencesEditor.isVisible
-                            }
-                        },
-                    )
-                },
-            )
+    }
+
+    private val aboutDialog by lazy {
+        jFrame("About Kindling", 300, 200, initiallyVisible = false) {
+            defaultCloseOperation = JFrame.HIDE_ON_CLOSE
+            isResizable = false
+            isUndecorated
+            type = Window.Type.UTILITY
+
+            contentPane = JLabel(
+                "Kindling v${System.getProperty("app.version")}",
```

Comment:
Will be automatically set at runtime by Conveyor:
https://conveyor.hydraulic.dev/10.1/configs/jvm/#keys